### ISSUE TITLE
Staging Review finding: Alert icons are not conveyed to assistive technology

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#122540]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Reapers Team
**Product:** VRE Ch31 Eligibility & Entitlement
**Feature:** VRE Ch31 Eligibility & Entitlement

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** Yes
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

The warning icon as part of the ineligible alert is not conveyed to assistive technology. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="674" height="188" alt="alert icon" src="https://github.com/user-attachments/assets/c669c7ee-1b8c-4c98-826a-fa59a6c43f29" />

</details>

## Recommended action

Meaning is being conveyed visually, but not programmatically, making the content unavailable to assistive technology. As sighted users, we build context of the alert via the icon and the background color, indicating that this information is important. Screen reader users aren't given an equivalent experience, as the icon is hidden from screen readers. Ensure that the icon is exposed is assistive technology by implementing a text alternative. 

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 1.1.1 Non-text Content - Level A](https://www.w3.org/TR/WCAG22/#non-text-content)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jeana-adhoc** [comment]
The DST has created new tickets for the work to update the components to address this finding
- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/5046
